### ISSUE TITLE
Starting Hadolint Container Fails with exec: "tail": executable file not found in $PATH: unknown

### ISSUE THREAD

**Issue by atalwar-sephora** [hadolint/hadolint#611]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
I am trying to start hadolint container using below command

`docker create --name 9a37a772568f45f2ba181180907608fc_hadolinthadolint_6be07c  --workdir /__w/docker-base-images/docker-base-image --entrypoint "tail" hadolint/hadolint "-f" "/dev/null" `

`docker start 9a37a772568f45f2ba181180907608fc_hadolinthadolint_6be07c`

But It fails with below error:

`Error response from daemon: OCI runtime create failed: container_linux.go:370: starting container process caused: exec: "tail": executable file not found in $PATH: unknown
Error: failed to start containers: 9a37a772568f45f2ba181180907608fc_hadolinthadolint_6be07c`

### Actual behavior
the container should start without showing error on `tail` not found.

### Steps to reproduce the behavior
`docker create --name 9a37a772568f45f2ba181180907608fc_hadolinthadolint_6be07c  --workdir /__w/docker-base-images/docker-base-image --entrypoint "tail" hadolint/hadolint "-f" "/dev/null" `

`docker start 9a37a772568f45f2ba181180907608fc_hadolinthadolint_6be07c`

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v2.2.0-0-ga34e5f3
```

**Dockerfile** (if relevant)

```Dockerfile
Dockerfile
```

**Additional environment details (OS, stack version, etc.)**
ubuntu-20.04

**Comment by lorenzo** [comment]
The docker image does not have a shell or any shell tools. Try using the Debian based image instead

**Comment by rittneje** [comment]
@lorenzo In my experience such docker images usually include "distroless" in the tag to make it clear (e.g., hadolint/hadolint:v2.3.0-distroless).

**Comment by atalwar-sephora** [comment]
@lorenzo  Thanks, it works with debian image. I used `hadolint/hadolint:latest-debian`

**Comment by atalwar-sephora** [comment]
Use Debian or alpine base image for shell.
